Sources/Plasma/PubUtilLib/plScene/plSceneNode.cpp

```diff
@@ -141,14 +141,14 @@ void plSceneNode::Harvest(plVolumeIsect* isect, hsTArray<plDrawVisList>& levList
         fDrawPool[idx]->GetSpaceTree()->HarvestLeaves(isect, visSpans);
         if (!visSpans.empty())
         {
-            plDrawVisList* drawVis = levList.Push();
-            drawVis->fDrawable = fDrawPool[idx];
-            drawVis->fVisList.swap(visSpans);
+            plDrawVisList& drawVis = levList.emplace_back();
+            drawVis.fDrawable = fDrawPool[idx];
+            drawVis.fVisList.swap(visSpans);
```

Comment:
This could be another construct in-place, but it would defeat the little visSpan "optimization" Cyan has going.